src/styles/index.css

```diff
@@ -0,0 +1,270 @@
+@charset "UTF-8";
+
+iframe {
+  display: none;
+}
+
+body {
+  min-width: 230px;
+  max-width: 550px;
+  margin: 0 auto;
+}
+
+.notification {
+  transition-property: opacity, min-height;
+  transition-duration: 1s;
+  min-height: 36px;
+}
+
+.notification.hidden {
+  min-height: 0;
+  opacity: 0;
+  pointer-events: none;
+}
+
+.todoapp {
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-size: 24px;
+  font-weight: 300;
+  color: #4d4d4d;
+  margin: 40px 20px;
+}
+
+.todoapp__content {
+  margin-bottom: 20px;
+  background: #fff;
+  box-shadow:
+    0 2px 4px 0 rgba(0, 0, 0, 0.2),
+    0 25px 50px 0 rgba(0, 0, 0, 0.1);
+}
+
+.todoapp__title {
+  font-size: 100px;
+  font-weight: 100;
+  text-align: center;
+  color: rgba(175, 47, 47, 0.15);
+  -webkit-text-rendering: optimizeLegibility;
+  -moz-text-rendering: optimizeLegibility;
+  text-rendering: optimizeLegibility;
+}
+
+.todoapp__header {
+  position: relative;
+}
+
+.todoapp__toggle-all {
+  position: absolute;
+  height: 100%;
+  width: 45px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 24px;
+  color: #e6e6e6;
+  border: 0;
+  background-color: transparent;
+  cursor: pointer;
+}
+
+.todoapp__toggle-all.active {
+  color: #737373;
+}
+
+.todoapp__toggle-all::before {
+  content: '❯';
+  transform: translateY(2px) rotate(90deg);
+  line-height: 0;
+}
+
+.todoapp__new-todo {
+  width: 100%;
+  padding: 16px 16px 16px 60px;
+  font-size: 24px;
+  line-height: 1.4em;
+  font-family: inherit;
+  font-weight: inherit;
+  color: inherit;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  border: none;
+  background: rgba(0, 0, 0, 0.01);
+  box-shadow: inset 0 -2px 1px rgba(0, 0, 0, 0.03);
+}
+
+.todoapp__new-todo::-moz-placeholder {
```

Comment:
If you really need it disable the rule:

![image](https://github.com/user-attachments/assets/22b6b929-39d7-4611-9e77-bda355a550c0)